dopamine/discrete_domains/run_experiment/helper.py

```diff
@@ -0,0 +1,107 @@
+# coding=utf-8
```

Comment:
I think there's a small cost in each additional source file (in this case, taking functionality out of `run_experiment.run_experiment` and putting it into `run_experiment.helper`.  What do we get in return?

Reply:
it's to avoid circular imports.

`run_async_training` needs `Runner` in `run_experiment`. And `create_runner` needs `AsyncRunner` in `run_async_training`. Until now everything was in the same file. Since I don't want to add `AsyncRunner` to the same file I also need to move `create_runner` to a separate file.

Reply:
Moved async runner to same file as other runners.